page02_playlist.php

```diff
@@ -1,79 +1,120 @@
+<?php
```

Comment:
same changes as 01_query and 02_query and change in positioning - copy and pasted but php code changed as well